mmv1/products/appengine/api.yaml

```diff
@@ -1462,3 +1462,57 @@ objects:
             required: true
             description: |
               Mapping from version IDs within the service to fractional (0.000, 1] allocations of traffic for that version. Each version can be specified only once, but some versions in the service may not have any traffic allocation. Services that have traffic allocated cannot be deleted until either the service is deleted or their traffic allocation is removed. Allocations must sum to 1. Up to two decimal place precision is supported for IP-based splits and up to three decimal places is supported for cookie-based splits.
+  - !ruby/object:Api::Resource
+    name: 'ServiceNetworkSettings'
+    description: |
+      A NetworkSettings resource is a container for ingress settings for a version or service.
+    base_url: 'apps/{{project}}/services'
+    self_link: 'apps/{{project}}/services/{{service}}'
+    create_url: 'apps/{{project}}/services/{{service}}?updateMask=networkSettings'
+    create_verb: :PATCH
+    update_url: 'apps/{{project}}/services/{{service}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'appengine#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/global/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'service'
```

Comment:
I think technically yes? since the API call the terraform resource makes is only a PATCH - This is another one I took from the SplitTraffic resource since they are very similar, but not really sure on the correct behaviour here